superdesk/metadata/item.py

```diff
@@ -374,6 +374,10 @@
         'type': 'list',
         'schema': Resource.rel('highlights', True)
     },
+    'marked_desks': {
```

Comment:
I'm just wondering if this should be list of ids, or if it would make sense to add some more metadata to each desk (like if this was seen by someone etc), just thinking ahead so we wouldn't have to migrate such data